README.md

```diff
@@ -222,7 +274,22 @@ stringData:
   tokenurl: "<auth_url>"
   tokenurlsuffix: "/oauth/token"
 ```
+mTLS Access Credentials
 
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sap-btp-service-operator
+  namespace: sap-btp-operator
+type: kubernetes.io/tls
+stringdata:
+  clientid: <clientid>
+  certificate: <certificate>
+  key: <key>
+  sm_url: <sm_url>
+  tokenurl: <auth_url>
+```
```

Comment:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: sap-btp-service-operator
  namespace: sap-btp-operator
type: Opaque
stringdata:
  clientid: <clientid>
  tls.crt: <certificate>
  tls.key: <key>
  sm_url: <sm_url>
  tokenurl: <auth_url>
  tokenurlsuffix: "/oauth/token"
```

Reply:
@I065450 I wanted to be sure the example was ok before I added it here too.

Reply:
@I065450 changed.